woocommerce/woocommerce
# Fix for issue woocommerce#20975

### All Submissions:






### Changes proposed in this Pull Request:



Making the shipping address blank instead of assigning the billing address to shipping address, when customer doesn't tick the "add a different shipping address" checkbox.

Closes #20975 

### How to test the changes in this Pull Request:

1. In the checkout page, don't check the "use a different address for shipping".
2. Refresh the page several time.
3. Tick the checkbox and the value will be the shipping address of the customer, instead of the billing address.

### Other information:




### Changelog entry

> Enter a short summary of all changes on this Pull Request. This will appear in the changelog if accepted.
